jipp-pdl/src/main/kotlin/com/hp/jipp/pdl/pwg/PwgSettings.kt

```diff
@@ -24,7 +24,10 @@ data class PwgSettings @JvmOverloads constructor(
     val sheetBack: String = PwgRasterDocumentSheetBack.normal,
 
     /** The orientation used when printing the current job. */
-    val orientation: Orientation = Orientation.portrait
+    val orientation: Orientation = Orientation.portrait,
+
+    /** The jobPagesPerSet job attribute if used when printing the current job. */
+    val jobPagesPerSet: Int? = null
```

Comment:
Instead of "jobPagesPerSet" which is a value we're not really using, let's just pass through the request to allowPadding or not.

```kotlin
/**
 * When true, adds padding for duplex odd-page-count jobs.
 *
 * Set this to false if, for example, the printer supports and received `job-pages-per-set` because it already knows where to add blank pages.
 */
val allowPadding: Boolean = true
```